source/module_esolver/esolver_fp.h

```diff
@@ -59,6 +59,15 @@ namespace ModuleESolver
         //! K points in Brillouin zone
         K_Vectors kv;
 
+#ifdef __LCAO
```

Comment:
Please don't add new functions in ESolver_FP

Reply:
init_orb has been removed from ESolver_FP and placed in esolver/init_orb.hpp.